### ISSUE TITLE
[feature] play indicator that doesn't block album art

### ISSUE THREAD

**Issue by aerickson** [Soundnode/soundnode-app#879]
I enjoy looking at album art and it's really hard to see with the current way the app indicates the track is being played. Maybe a smaller icon or perhaps the bottom bar is enough of an indicator. Example included below.

![soundnode](https://cloud.githubusercontent.com/assets/47889/19004967/27901fca-870d-11e6-9f7c-097f8659b5a2.png)

Thanks for making Soundnode! It's awesome. :)

**Comment by weblancaster** [comment]
@Pitros I believe you added this while ago.. I also agree with @aerickson we don't need to display if the song is playing or not on the album.. I'm going to remove.

**Comment by Pitros** [comment]
@weblancaster no, that wasn't me. But I agree and you can remove it, or just move it somewhere.

**Comment by weblancaster** [comment]
Oh ok my bad @Pitros 

yes I'm just going to remove it
